updateCertificationStatusApp.sh

```diff
@@ -5,4 +5,20 @@
 # 15 5 * * * cd /tempApps && ./updateCertificationStatusApp.sh
 # This will run it at 0515 UTC, which (depending on DST) is 0015 EST
 
-java -jar certStatusApp-jar-with-dependencies.jar
\ No newline at end of file
+# create timestamp and filename
+TIMESTAMP=$(date "+%Y.%m.%d-%H.%M.%S")
+log=logs/log.certStatusApp.$TIMESTAMP.txt
```

Comment:
As long as the directory is there on the machine that's going to run it that's fine. See if you can figure out why git doesn't track the directory though; it's a good thing to know